### ISSUE TITLE
"Implicit braces are forbidden" triggered with constant in class body

### ISSUE THREAD

**Issue by nathan-muir** [clutchski/coffeelint#324]
Reproduction similar to #68 

```coffeescript
class ABC

  @CONST_1 = 'test'

  constructor: (abc)->
    @def = abc * 2

  getDef: -> @def
```

```json
{
    "no_implicit_braces": {
        "level": "error", 
        "strict": false
    }
}
```

output:
```
[~]$ coffee --version
CoffeeScript version 1.6.3
[~]$ coffeelint --version
1.5.5
[~]$ coffeelint -f coffeelint.cfg badclass.coffee
  ✗ badclass.coffee
     ✗ #5: Implicit braces are forbidden.

✗ Lint! » 1 error and 0 warnings in 1 file
```

**Comment by murphybob** [comment]
+1 this - it is causing some erroneous linting errors on our build we can't get rid of as the code is valid, so we've had to turn the rule off which is a pain.  As a note we're using strict: false.

**Comment by lookfirst** [comment]
:+1:

**Comment by ozankasikci** [comment]
i'm not entirely sure if this is fixed because this code which is in a class body gives me the same error "Implicit braces are forbidden.".

`OPTIONS =
    type            : 'type'
    autoDelete  : yes

  constructor: (@owner, options = {}) ->`

**Comment by swang** [comment]
This is already addressed in #484

**Comment by ozankasikci** [comment]
awesome :+1:
